neointerface/neointerface.py

```diff
@@ -1831,15 +1831,21 @@ def import_json_data(self, json_str: str):
         """
 
         try:
-            json_list = json.loads(json_str)  # Turn the string (representing a JSON list) into a list
+            json_obj = json.loads(json_str)  # Turn the string (representing a JSON list) into a list
         except Exception as ex:
             raise Exception(f"Incorrectly-formatted JSON string. {ex}")
 
+        if isinstance(json_obj, list):
+            json_list = json_obj
+        elif isinstance(json_obj, dict):
```

Comment:
I dont think this case would ever happen with the code as it is now, as if json_obj is a dict it fails on line 1834 
![image](https://github.com/GSK-Biostatistics/neointerface/assets/106977606/f5023c50-c934-4c7b-84e2-19f784017b59)

Reply:
this means you currently cant use the output of export_dbase_json as an input for this